src/main/java/io/specto/hoverfly/junit/api/view/ResponseDiffForRequestView.java

```diff
@@ -0,0 +1,44 @@
+package io.specto.hoverfly.junit.api.view;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import java.util.stream.IntStream;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ResponseDiffForRequestView {
+    private final SimpleRequestDefinitionView request;
+    private final List<DiffReport> diffReports;
+
+    @JsonCreator
+    public ResponseDiffForRequestView(
+        @JsonProperty("request") SimpleRequestDefinitionView request,
+        @JsonProperty("diffReports") List<DiffReport> diffReports) {
+        this.request = request;
+        this.diffReports = diffReports;
+    }
+
+    public String createDiffMessage() {
+        StringBuilder message = new StringBuilder()
+            .append("\nFor the request with the simple definition:\n" + getRequest().toString())
+            .append("\n\nhave been recorded " + getDiffReports().size() + " diff(s):\n");
+
+        IntStream.range(0, getDiffReports().size())
```

Comment:
You are using for loop for the other `createDiffMessage`, could you keep the style consistent please? I'd prefer a zero-based for loop for simple iteration, unless you need other streaming feature such as `map` to string and then collect using `Collector.joining` to create the report.